src/Tribe/Main.php

```diff
@@ -667,15 +667,23 @@ public function add_help_section_extra_content( $help ) {
 	/**
 	 * Register Event Tickets with the template update checker.
 	 *
+	 * @since TBD Updated template structure.
+	 *
 	 * @param array $plugins
 	 *
 	 * @return array
 	 */
 	public function add_template_updates_check( $plugins ) {
 		$plugins[ __( 'Event Tickets', 'event-tickets' ) ] = [
-			self::VERSION,
-			$this->plugin_path . 'src/views/tickets',
-			trailingslashit( get_stylesheet_directory() ) . 'tribe-events/tickets',
+				self::VERSION,
```

Comment:
:no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16

<sub>hash: c7ec9fb4f14b5f5f4216f4c8aead59d9</sub>